tests/test_cli.py

```diff
@@ -89,9 +90,8 @@ def run_successful(self, result: Result, obj: Any) -> None:
         if result.exit_code:
             raise RuntimeError(f"{obj} failed") from result.exception
 
-    def run_convert(self, runner: CliRunner, test_case: SSSOMTestCase) -> Result:
+    def run_convert(self, runner: CliRunner, test_case: SSSOMTestCase, fmt="owl") -> Result:
         """Run the convert test."""
-        fmt = "owl"
```

Comment:
Updated `run_convert` to accommodate different output formats. This will also be used by #285 to run `fhir_json`.